Firebase/Core/FIRApp.m

```diff
@@ -471,40 +471,45 @@ + (NSError *)errorForInvalidAppID {
                          userInfo:errorDict];
 }
 
-+ (void)registerAsConfigurable:(Class<FIRCoreConfigurable>)klass {
++ (void)registerAsConfigurable:(Class<FIRLibrary>)klass {
```

Comment:
I think this call could probably go away and the logic could go into `registerLibrary:`, that way it's only one call for the library. Or would you rather keep the logic separate?